### ISSUE TITLE
Window title incorrect on Windows

### ISSUE THREAD

**Issue by beewyka819** [emoon/minifb#57]
On Windows 10, when calling mfb_open_ex with a title of "Hello World!", the title for the resulting window is completely messed up. I THINK windows expects some UTF-16 encoding. If so, this might be the culprit
 if minifb doesn't perform this conversion, however I'm not familiar with winapi stuff so I could be wrong.

Here's what "Hello World!" becomes:
![image](https://user-images.githubusercontent.com/8059789/113234917-3817ac00-9270-11eb-9b79-9cdc7a66b11d.png)

**Comment by Darky-Lucera** [comment]
Windows has two modes unicode and multibyte. Be sure to use multibyte while compiling.

In unicode it uses wchar_t for strings that is (IMHO) a source of problems and incompatibilities in general

**Comment by beewyka819** [comment]
I tried compiling with multibyte. Let me try again and see if I missed anything, and I'll get back to you

**Comment by beewyka819** [comment]
Yeah, when compiling with multibyte the issue persists. I have a staticlib handling the windowing stuff, and a testing app using it. Before I forgot to change encoding on the test app, but now I've switched both over to multibyte, however the issue persists.

**Comment by Darky-Lucera** [comment]
And, have the examples an incorrect window title?
Because I checked all of them and they work.
I tried MinGW and VS 2019 (Windows 10).

Give me some more info about your project

**Comment by beewyka819** [comment]
Ah, I've found the issue. I'm using premake for generating project files, and when I do, it resets the mode to unicode. While I made sure to undo this on my projects, I forgot to undo it for minifb's project settings. That seemed to fix it.
